### PR TITLE
Documentation for `netapp-trident` template

### PR DESCRIPTION
**Pull request by balaramesh**
Per Neeraj's changes:

There will now exist 3 templates for Trident:

1. Install Trident
   1.1 Create an ONTAP-NAS backend
   1.2 Create an ONTAP-SAN backend.

This PR includes documentation for the `netapp-trident` template

### CODE REVIEW

File: config-templates/netapp/netapp-ontap-nas/20.07/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+   * You must have a dedicated SVM for Trident. Volumes that are created by Trident are created in this SVM.
+   * You must have one or more aggregates assigned to the SVM.
+   * You must have one or more dataLIFs for the SVM. Depending on the protocol used (NFS/iSCSI), at least one dataLIF is required.
+   * You must have NFS services enabled on the SVM.
+   * You must set up a snapshot policy on the SVM.
+* Share following details with Location Admin
+Review the template parameters and retrieve the values from your NetApp cluster.
+
+**Planning consideration for Location Admin**
+* Make sure `netapp-trident` is deployed on the taregt cluster
```
Comment (derekpoindexter) [inline config-templates/netapp/netapp-ontap-nas/20.07/README.md:18]:
```suggestion
* Make sure that the `netapp-trident` template is deployed on the target clusters.
```

File: config-templates/netapp/netapp-ontap-nas/20.07/custom-parameters.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    "obfuscate": "true"
+  }, {
+    "description": "The NAS option for the NFS export policy.",
+    "displayname": "Export Policy",
+    "name": "exportPolicy",
+    "default": "default",
+    "required": "false",
+    "category": "config"
+  }, {
+    "description": "Maximum requestable volume size and qtree parent volume size",
```
Comment (derekpoindexter) [inline config-templates/netapp/netapp-ontap-nas/20.07/custom-parameters.json:41]:
```suggestion
    "description": "The maximum requestable volume size and qtree parent volume size.",
```

File: config-templates/netapp/netapp-ontap-nas/20.07/custom-parameters.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    "displayname": "Limit Volume Size",
+    "name": "limitVolumeSize",
+    "default": "50Gi",
+    "required": "false",
+    "category": "config"
+  }, {
+    "description": "Fail provisioning if usage is above this percentage",
```
Comment (derekpoindexter) [inline config-templates/netapp/netapp-ontap-nas/20.07/custom-parameters.json:48]:
```suggestion
    "description": "Provisioning fails if usage is above this percentage.",
```

File: config-templates/netapp/netapp-ontap-nas/20.07/README.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+```
+
+```
+oc get sc | grep sat
+```
+
+
+## Reference
+
+- https://netapp-trident.readthedocs.io/en/stable-v20.07/kubernetes/operations/tasks/backends/ontap/ontap-nas/index.html
```
Comment (derekpoindexter) [inline config-templates/netapp/netapp-ontap-nas/20.07/README.md:87]:
```suggestion
- [NetApp documentation](https://netapp-trident.readthedocs.io/en/stable-v20.07/kubernetes/operations/tasks/backends/ontap/ontap-nas/index.html)
```

File: config-templates/netapp/netapp-ontap-nas/20.07/README.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+- Support: https://netapp-trident.readthedocs.io/en/stable-v20.10/support/support.html
```
Comment (derekpoindexter) [inline config-templates/netapp/netapp-ontap-nas/20.07/README.md:88]:
```suggestion
- [Support documentation](https://netapp-trident.readthedocs.io/en/stable-v20.10/support/support.html)
```

**Comment by nkkashyap** [comment]
Stale

**Pull request closed without merging**